README.md

```diff
@@ -199,47 +226,54 @@ In a **classic application** this could look like this:
 private OutboxService outboxService;
 
 @Transactional
-public void doSomething(String id, String name) {
+public void doSomething(String id,String name){
 
-    // Here s.th. else would be done within the transaction, e.g. some entity created.
+        // Here s.th. else would be done within the transaction, e.g. some entity created.
 
-    SomeEvent event = SomeEvent.newBuilder()
-            .setId(id)
-            .setName(name)
-            .build();
-    outboxService.saveForPublishing("some-topic", id, event);
-}
+        SomeEvent event=SomeEvent.newBuilder()
+        .setId(id)
+        .setName(name)
+        .build();
+        outboxService.saveForPublishing("some-topic",id,event);
+        }
 ```
 
-In a **reactive application** it would look like this (you could also use `@Transactional` if you'd prefer this rather than using the `TransactionalOperator`):
+In a **reactive application** it would look like this (you could also use `@Transactional` if you'd prefer this rather
+than using the `TransactionalOperator`):
 
 ```java
 @Autowired
 private OutboxService outboxService;
 @Autowired
 private TransactionalOperator rxtx;
 
-public Mono<OutboxRecord> doSomething(String name) {
+public Mono<OutboxRecord> doSomething(String name){
 
-    // Here s.th. else would be done within the transaction, e.g. some entity created.
-    return createSomeThing(name)
-        .flatMap(someThing -> {
-            SomeEvent event = SomeEvent.newBuilder()
-                .setId(someThing.getId())
-                .setName(someThing.getName())
-                .build();
-            return outboxService.saveForPublishing("some-topic", someThing.getId(), event);
+        // Here s.th. else would be done within the transaction, e.g. some entity created.
+        return createSomeThing(name)
+        .flatMap(someThing->{
+        SomeEvent event=SomeEvent.newBuilder()
+        .setId(someThing.getId())
+        .setName(someThing.getName())
+        .build();
+        return outboxService.saveForPublishing("some-topic",someThing.getId(),event);
         })
         .as(rxtx::transactional);
 
-}
+        }
 ```
```

Comment:
I think now the code formatting is broken. Can you fix it? 🤓

Reply:
Should be fixed now -- could you confirm? 🙂

Reply:
The formatting looks still not correct for me 😕 

For example here:
```
@Transactional
public void doSomething(String id,String name){

        // Here s.th. else would be done within the transaction, e.g. some entity created.

        SomeEvent event=SomeEvent.newBuilder()
        .setId(id)
        .setName(name)
        .build();
        outboxService.saveForPublishing("some-topic",id,event);
        }
```
I'd like to have it formatted like:
```
@Transactional
public void doSomething(String id, String name) {

    // Here s.th. else would be done within the transaction, e.g. some entity created.

    SomeEvent event = SomeEvent.newBuilder()
            .setId(id)
            .setName(name)
            .build();
    outboxService.saveForPublishing("some-topic", id, event);
}
```

Reply:
Would you mind taking another look? I've reverted all formatting changes (except from a selected few that I think are really beneficiary, like a blank line between two methods).